addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -83,11 +83,20 @@ $o-wslides-fs-side-width: 300px;
         }
     }
 
-    .o_wsildes_quiz_question_input {
-        .o_wslides_js_quiz_is_correct > label,
-        .o_wslides_js_quiz_add_answer,
-        .o_wslides_js_quiz_remove_answer {
+    .o_wslides_js_lesson_quiz {
+        i.o_wslides_js_quiz_icon {
             cursor: pointer;
+            color: #7d757d;
```

Comment:
Same here, it is better to use variables or existing classes. It seems a text-muted should do the trick here.